website/source/docs/platform/k8s/injector/index.html.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: "docs"
+page_title: "Agent Sidecar Injector Overview"
+sidebar_current: "docs-platform-k8s-injector"
+sidebar_title: "Agent Injector"
+description: |-
+  The Vault Agent Sidecar Injector is a Kubernetes admission webhook that adds Vault Agent containers to pods for consuming Vault secrets.
+---
+
+# Agent Sidecar Injector
+
+The Vault Agent Injector alters pod specifications to include Vault Agent
+containers that render Vault secrets to a shared memory volume using 
+[Vault Agent templates](/docs/agent/template/index.html).
+By rendering secrets to a shared volume, containers within the pod can consume 
+Vault secrets without being Vault aware.
+
+The injector is a [Kubernetes Mutation Webhook Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/).
+The controller intercepts pod events and applies mutations to the pod if annotations exist within
+the request.  This functionality is provided by the [vault-k8s](https://github.com/hashicorp/vault-k8s)
+project and can be automatically installed and configured using the
+[Vault Helm](https://github.com/hashicorp/vault-helm) chart.
+
+## Overview
+
+The Vault Agent Injector works by intercepting pod `CREATE` and `UPDATE`
+events in Kubernetes.  The controller parses the event and looks for the metadata
+annotation `vault.hashicorp.com/agent-inject: true`.  If found, the controller will
+alter the pod specification based on other annotations present.
+
+### Mutations
+
+At a minimum, every container in the pod will be configured to mount a shared
+memory volume.  This volume, mounted to `/vault/secrets`, will be used by the Vault
+Agent containers for sharing secrets with the other containers in the pod.
+
+Next, two types of Vault Agent containers  can be injected: init and sidecar.  The
+init container will prepopulate the shared memory volume with the requested
+secrets prior to the other containers starting.  The sidecar container will
+continue to authenticate and render secrets to the same location as the pod runs.
+Using annotations, the intialization and sidecar containers may be disabled.
+
+Last, two addtional types of volumes can be optionally mounted to the Vault Agent
+containers.  The first is secret volume containing TLS Client certificate/key and
```

Comment:
There are two instances of "/key" here. Suspect these need to be removed.

Reply:
Improved this a bit.